cares_reinforcement_learning/networks/NaSATD3/actor.py

```diff
@@ -5,11 +5,19 @@
 
 
 class Actor(nn.Module):
-    def __init__(self, latent_size: int, num_actions: int, encoder: nn.Module):
+    def __init__(
+        self,
+        latent_size: int,
+        num_actions: int,
+        encoder: nn.Module,
+        hidden_size: list[int] = None,
+    ):
         super().__init__()
+        if hidden_size is None:
+            hidden_size = [1024, 1024]
```

Comment:
author said 256 should be enough, but both are fine

Reply:
NaSATD3? Currently just using what David had put in originally - given the other image based stuff is 1024 it kind of makes sense to keep it as 1024 - SACAE and TD3AE